src/Umbraco.Web.UI.Client/src/common/directives/components/grid/grid.rte.directive.js

```diff
@@ -13,6 +13,10 @@ angular.module("umbraco.directives")
                 // TODO: A lot of the code below should be shared between the grid rte and the normal rte
 
                 var promises = [];
+                
+                var d = new Date();
+                var n = d.getTime();
+                scope.textAreaHtmlId = scope.uniqueId + "_" + n + "_rte";
```

Comment:
We have a method to create guids which is probably nicer than this, you can do `String.CreateGuid()`

Reply:
Great insight, thanks.

Basically this fixes so TinyMCE gets a unique ID. The code is copied from the TinyMCE-PropertyEditor, where someone have implemented this fix. So I have made sure we are using the same fix in the Grid.

But I definitely agree that it would be better to use String.CreateGuid()